bridgesync/bridgesync.go

```diff
@@ -328,6 +328,16 @@ func (s *BridgeSync) GetClaimsPaged(
 	return s.processor.GetClaimsPaged(ctx, page, pageSize, networkIDs, globalIndex)
 }
 
+func (s *BridgeSync) GetUnsetClaimsPaged(
+	ctx context.Context,
+	page, pageSize uint32, networkIDs []uint32, globalIndex *big.Int) ([]*UnsetClaim, int, error) {
```

Comment:
Unused `networkIDs` param, remove it?
```suggestion
	page, pageSize uint32, globalIndex *big.Int) ([]*UnsetClaim, int, error) {
```